### DISCUSSION TITLE
Single dose freezing: store drawer number in beans

### DISCUSSION THREAD

**Discussion by reox** [graphefruit/Beanconqueror#651] [Ideas]
Some time ago, I started to freeze beans in single doses. Now, my freezer has collected many different beans I I often have to check that I use the right one (I know, first world problems...).
I have an idea that might be neat to implement: Add another layer to beans and allow to virtually store them in drawers or boxes, using numbers or colors (or whatever floats your boat).
The idea would be then, that you can go into the app, find a bean you want to brew, lookup the drawer/box where it is in and find it.

This is actually somewhat similar to the idea I had for NFC tags (see [#504 )](https://github.com/graphefruit/Beanconqueror/issues/504#issuecomment-1478200130))

Maybe there is even a better way to do this. What do you think? How do you handle that?

**Comment by graphefruit** [comment]
Hey @reox,
on the Beanconqueror Discord there is a thread for freezing beans.
Please join, and have a look here: https://discord.com/channels/1095806336907874347/1181237669355855943
There was already a lot of discussion

**Reply by reox** [reply]
Found out yesterday that you can show the notes on the bean overview page. I simply added in which box I put the beans now in the notes - which is more than sufficient for what I need!

**Reply by graphefruit** [reply]
The freezing feature will be live with Version 7.4.
